app/main.py

```diff
@@ -1,3 +1,42 @@
-def shop_trip():
-    # write your code here
-    pass
+import os
+import json
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+
+
+def shop_trip() -> None:
+    config_path = os.path.join(os.path.dirname(__file__), "config.json")
+    with open(config_path, "r") as file:
+        config = json.load(file)
+    fuel_price = config["FUEL_PRICE"]
+    shops = [Shop(shop_data["name"], tuple(shop_data["location"]),
+                  shop_data["products"]) for shop_data in config["shops"]]
```

Comment:
```suggestion
    shops = [Shop(**shop_data) for shop_data in config["shops"]]
```
you can try using unpacking here